kong.client.loadConsumer(UUID) throws error in custom js Plugin

It appears an RPC error is being thrown when trying to load a consumer by UUID.

We have verified our configurations and verified we are correctly passing in the UUID for our anonymous consumer, but continue to receive the following error:

**_[ERROR] rpc: #13 method "StepConsumer" not implemented_**

This is for a custom authorizer plugin.  Using references to the lua plugins for authorization, we are attempting to call **kong.client.loadConsumer(UUID)** inside the **access** method.

--- a/server.js
+++ b/server.js
@@ -10,7 +10,7 @@ const {
 const PipePair = require(path.join(__dirname, 'lib', 'pipe'))
 const PDK = require(path.join(__dirname, 'pdk'))
 
-const entities = ('Service', 'Consumer', 'Route', 'Plugin', 'Credential', 'MemoryStats')
+const entities = ['Service', 'Consumer', 'Route', 'Plugin', 'Credential', 'MemoryStats']
 const MSG_RET = 'ret'
 
 class PluginServerError extends Error {

